ios/FluentUI/Core/Theme/Tokens/AliasTokens.swift

```diff
@@ -153,6 +154,12 @@ public final class AliasTokens: NSObject {
         case presenceDnd
         case presenceAvailable
         case presenceOof
+
+        // Shared colors - Stroke
+        case dangerStroke1
+        case successStroke1
+        case warningStroke1
+        case severeStroke1
```

Comment:
Optional - to align with Figma could group the danger/warning/success/severe stroke colors with their foreground.background colors above. Makes it a little easier to navigate in the future when the ordering matches.